### PR TITLE
solution

### PR DESCRIPTION
**Pull request by RomSmile**

### CODE REVIEW

**Review by illiarerun — APPROVED** [review]
Good job 🔥

File: src/transformState.js [hunk 1]
```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const ourObject of actions) {
```
Comment (danheim) [inline src/transformState.js:8]: incorrect variable name

File: src/transformState.js [hunk 2]
```diff
@@ -5,7 +5,27 @@
...
+    switch (ourObject.type) {
+      case 'addProperties':
+        Object.assign(state, ourObject.extraData);
+        break;
+
+      case 'removeProperties':
+        for (let i = 0; i < ourObject.keysToRemove.length; i++) {
```
Comment (danheim) [inline src/transformState.js:15]: use for of

**Review by danheim — CHANGES_REQUESTED** [review]

File: src/transformState.js [hunk 3]
```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const ourAcrion of actions) {
```
Comment (danheim) [inline src/transformState.js:8]: our? are you from the soviet union?
```suggestion
  for (const action of actions) {
```

**Review by danheim — APPROVED** [review]

**Pull request closed without merging**